Library/Homebrew/cmd/bottle.rb

```diff
@@ -192,10 +199,19 @@ def bottle_formula(f)
 
         keg.delete_pyc_files!
 
+        tab = Tab.for_keg(keg)
+        tab["HEAD"] = formula_last_commit_hash
+        tab["time"] = formula_last_commit_time.to_i
+        tab.write
```

Comment:
Simply abandoning current HEAD attribute, which stores the information about what core code is used to build, can be a little tricky. The reason is we may introduce new core code which will generate different content. Therefore, core code version matters in reproducible bottles. 

Therefore, I suggest to keep current HEAD attribute as it is. In all, I think we should design tab file as such:
```
HEAD: homebrew core commit used for build. 
taps: a dict to store tap name and their
         corresponding commits. Where we should  
         enumerate all taps for formula or direct
         dependences. 
time: formula last commit time. 
```

This also means we should create new command or `install` mode to checkout corresponding commit for reproducible verification